docs/l3/api_flow.md

```diff
@@ -525,4 +525,97 @@ Note: A demand can be commented on multiple times and it does not change the dem
   }
 ```
 
-Note: At any time after a `match2` is proposed, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using `POST /v1/match2/{id}/rejection`. Once a `match2` is rejected, it is permanently closed.
\ No newline at end of file
+Note: At any time after a `match2` is proposed, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using `POST /v1/match2/{id}/rejection`. Once a `match2` is rejected, it is permanently closed.
+
+## Cancelling match2
+
+1. Viewing all `match2` cancellation transactions can be performed by `memberA` or `memberB` by making **GET** request to `/v1/match2/{match2Id}/cancellation` endpoint. Takes a `match2id` as a parameter property. Match2 must be in `finalAcceptance` state.
+
+
+#### response example
+```js
+[
+  ...,
+  {
+    "id": "3ba8e8b9-1e7b-47de-bb91-fdcb81c41ed7",
+    "state": "inBlock",
+    "localId": "56261a42-e046-488a-875d-7ed9e93dde09",
+    "apiType": "match2",
+    "transactionType": "cancellation",
+    "submittedAt": "2023-09-26T10:34:17.030Z",
+    "updatedAt": "2023-09-26T10:34:18.043Z"
+  }
```

Comment:
If you are going to show that it is a list of items then you should show 2 cancellations as an example, instead of just 1.

Reply:
left a `...` and also it's an array... thought it's self explanatory. Added one more record.